server.js

```diff
@@ -1,49 +1,41 @@
-let categories = require('./my-app-react/src/data/categories.json')
-let products = require('./my-app-react/src/data/products.json')
+let dataService = require('./dataservice.js');
 
-const express = require('express')
-const app = express()
+const express = require('express');
+const app = express();
 
 app.get('/info', (req, res) => {
-    info = [
-        {"serverName": "localhost5000"},
+    let info = [
+        {"serverName": "localhost5001"},
         {"sampleName": "chinemerem-react"},
         {"serverVersion": "1.0.0"}
-    ]
+    ];
     res.json(info)
-})
+});
 
 app.get('/products/all', (req, res) => {
-    res.json(products)
-})
+    let productsMap = dataService.getCombinedProductMap();
+    res.send(productsMap);
+});
 
 app.get('/products/:id', (req, res) => {
-    let urlProductID = req.params.id
-    let productsLocal = products.products
-
-    for (productID in productsLocal) {
-        let product = productsLocal[productID];
-        if(urlProductID == product.id) {
-            res.json(product)
-        }
-    }
-
-})
+    let urlProductID = req.params.id;
```

Comment:
Can't this be const? Also, use [object destructuring.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)